database/migrations/sql/20180604_06_create_users.sql

```diff
@@ -19,6 +19,7 @@ CREATE TABLE user_account (
   deleted_at                       TIMESTAMP WITH TIME ZONE,
 
   CONSTRAINT user_account_pk                  PRIMARY KEY (user_account_id),
+  CONSTRAINT user_account_to_gender_fk        FOREIGN KEY (gender_id) REFERENCES gender ON DELETE CASCADE
```

Comment:
This says that if we delete a row in `gender` it'll delete all linked rows in `user_account`. Seems dangerous?

Reply:
Oops...